### PR TITLE
Kraken ea

### PR DESCRIPTION
**Pull request by buchkovm**
* [x] I have read the [guidelines for bioconda recipes](https://bioconda.github.io/guidelines.html).
* [x] This PR adds a new recipe.
* [ ] This PR updates an existing recipe.
* [ ] This PR does something else (explain below).

### CODE REVIEW

File: recipes/kraken-ea/build.sh [hunk 1]
```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+
+#export CPPFLAGS="-I$PREFIX/include"
+#export LDFLAGS="-L$PREFIX/lib"
+#export CPATH=${PREFIX}/include
+
+mkdir -p "$PREFIX/libexec" "$PREFIX/bin"
```
Comment (daler) [inline recipes/kraken-ea/build.sh]: I've not seen `libexec` before and I don't see it in the [FSH](https://en.wikipedia.org/wiki/Filesystem_Hierarchy_Standard). Would it be possible to use `$PREFIX/share/$PKG_NAME-$PKG_VERSION-$PKG_BUILDNUM` instead?

File: recipes/kraken-ea/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    - libexec/kraken-translate
+
+requirements:
+  build:
+    - perl
+    - zlib
+
+  run:
+    - zlib
+    - jellyfish 1.*
```
Comment (daler) [inline recipes/kraken-ea/meta.yaml]: I think the `*` notation is still reserved for special things like numpy. Does `jellyfish<2` work?

**Comment by buchkovm** [comment]
The above requests have been successfully incorporated into the build script.

**Comment by druvus** [comment]
What is special with this fork compared to the original kraken?

**Comment by buchkovm** [comment]
This fork has been modified to split the reads into different fastq files based on the taxonomic classification of the read. For example you could end up with one file containing bacterial reads and another containing human reads. The fork is being added in preparation to being the dependency for another tool that utilizes this custom functionality.

**Pull request merged**